include/af/image.h

```diff
@@ -1348,6 +1374,37 @@ extern "C" {
     */
     AFAPI af_err af_rgb2ycbcr(af_array* out, const af_array in, const af_ycc_std standard);
 #endif
+
+#if AF_API_VERSION >= 34
+    /**
+       C Interface for finding image moments
+
+       \param[out] out is an array containing the calculated moments
+       \param[in]  in is an array of image(s)
+       \param[moment] is the moment(s) to calculate
+       \return     ref AF_SUCCESS if the moment calculation is successful,
+       otherwise an appropriate error code is returned.
+
+       \ingroup image_func_moments
+    */
+    AFAPI af_err af_moments(af_array *out, const af_array in, const af_moment_type moment);
+#endif
+
+#if AF_API_VERSION >= 34
+    /**
+       C Interface for calculating image moment(s) of a single image
+
+       \param[out] out is  a pointer to the outputted moment(s)
```

Comment:
Change the phrasing to be clearer. Also mention user needs to have adequate space allocated.